.github/workflows/test-scripts.yml

```
@@ -0,0 +1,48 @@
+name: "Cross platform script test"
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - 'feature/**'
+  pull_request:
+
+jobs:
+  test-scripts:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, windows-2022]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Blocking: can you use commit hash ID and not the tags?

```suggestion
        uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
```